### PR TITLE
Dark Mode: Modals

### PR DESCRIPTION
**Pull request by darkwing**
Modals have hardcoded white colors, so I've transitioned colors for dark mode.
ResetAcct

Also includes QR reader styles:
QR

### CODE REVIEW

File: ui/components/app/modals/index.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@
     bottom: 0;
     left: 0;
     z-index: 1040;
-    background-color: #373a47;
+    background-color: #373a47; /* TODO: What color here? */
```
Comment (darkwing) [inline ui/components/app/modals/index.scss:38]: @georgewrmarshall @GuillaumeRx What color would be best here?

File: ui/components/app/modals/qr-scanner/index.scss [hunk 2]
```diff
@@ -67,16 +67,16 @@
 
     button:last-of-type {
       margin-right: 0;
-      background-color: #009eec;
+      background-color: var(--color-primary-default);
       border: none;
-      color: #fff;
+      color: var(--color-text-default);
```
Comment (GuillaumeRx) [inline ui/components/app/modals/qr-scanner/index.scss:72]:
```suggestion
      color: var(--color-primary-inverse);
```

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]

**Review by GuillaumeRx — DISMISSED** [review]

**Comment by darkwing** [comment]
I'm cool with the background color changes; I used `transparent` because it inherited the same background.

File: ui/components/app/modals/index.scss (hunk 1 above)
Comment (georgewrmarshall) [reply ui/components/app/modals/index.scss:38]: As it's a background color I would suggest `var(--color-overlay-alterantive)` `--color-text-default` shouldn't be used for any background-color really. I'm trying to find out how and where this is used.

Comment (georgewrmarshall) [reply ui/components/app/modals/index.scss:38]: Doesn't look like this is even being used but I think it should be and overlay. I would suggest updating it to
```suggestion
    background-color: var(--color-overlay-alterantive);
```

**Review by georgewrmarshall — COMMENTED** [review]
Looks good. I have one suggestion

**Comment by darkwing** [comment]
Updated!

**Review by georgewrmarshall — APPROVED** [review]

**Review by brad-decker — APPROVED** [review]
Changes look good to me.

**Pull request merged**